### DISCUSSION TITLE
当运行简悦 · 同步助手后，如发现某些快捷键「被劫持」的解决方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1640] [重要通知]
### 问题描述：

当运行简悦 · 同步助手后，如发现某些快捷键 **被劫持**，请使用下面的方式解决。

### 解决方案（Mac 用户）

请 [重新下载](http://ksria.com/simpread/docs/#/Sync?id=%e4%b8%8b%e8%bd%bd) 最新版的 Mac 版

###  解决方案（Window 用户）

解决方案请看 https://github.com/Kenshin/simpread/issues/1632#issuecomment-752308920

**Comment by here-ted** [comment]
您好, mac 系统的同步助手会默认占用command + N 用来打开稍后读,怎么禁用或者更改快捷键呢?只看到可以修改独立窗口打开的快捷键

**Reply by Kenshin** [reply]
解决方案 [看这里](https://simpread.pro/docs/#/%E7%A8%8D%E5%90%8E%E8%AF%BB%E5%BF%AB%E6%8D%B7%E9%94%AE)。

**Comment by zhongshanhouren** [comment]
您好，想用command + A 全选文字的时候会被聚焦阅读占用，如何禁用或者更改这个快捷键呢？您好，想用command + A 全选文字的时候会被聚焦阅读占用，如何禁用或者更改这个快捷键呢？
